test/lib/run-build-lambda.js

```diff
@@ -11,13 +12,21 @@ function runAnalyze(wrapper, context) {
 async function runBuildLambda(inputPath) {
   const inputFiles = await glob('**', inputPath);
   const nowJsonRef = inputFiles['vercel.json'] || inputFiles['now.json'];
-  expect(nowJsonRef).toBeDefined();
+
+  if (typeof expect !== 'undefined') {
```

Comment:
Is `expect` undefined in some cases? 🤔

Reply:
Added this to allow requiring this util outside of tests